R/pkg/R/mllib.R

```diff
@@ -647,6 +654,195 @@ setMethod("predict", signature(object = "KMeansModel"),
+#'                           If numClasses == 1 || numClasses == 2, set to "binomial".
+#'                           Else, set to "multinomial".
+#'                 - "binomial": Binary logistic regression with pivoting.
+#'                 - "multinomial": Multinomial logistic (softmax) regression without pivoting.
+#'                 Default is "auto".
+#' @param standardization whether to standardize the training features before fitting the model. The coefficients
+#'                        of models will be always returned on the original scale, so it will be transparent for
+#'                        users. Note that with/without standardization, the models should be always converged
+#'                        to the same solution when no regularization is applied. Default is TRUE, same as glmnet.
+#' @param threshold in binary classification, in range [0, 1]. If the estimated probability of class label 1
+#'                  is > threshold, then predict 1, else 0. A high threshold encourages the model to predict 0
+#'                  more often; a low threshold encourages the model to predict 1 more often. Note: Setting this with
+#'                  threshold p is equivalent to setting thresholds (Array(1-p, p)). When threshold is set, any user-set
+#'                  value for thresholds will be cleared. If both threshold and thresholds are set, then they must be
+#'                  equivalent. Default is 0.5.
+#' @param thresholds in multiclass (or binary) classification to adjust the probability of predicting each class.
+#'                   Array must have length equal to the number of classes, with values > 0, excepting that at most one
+#'                   value may be 0. The class with largest value p/t is predicted, where p is the original probability
+#'                   of that class and t is the class's threshold. Note: When thresholds is set, any user-set
+#'                   value for threshold will be cleared. If both threshold and thresholds are set, then they must be
+#'                   equivalent. Default is NULL.
+#' @param weightCol The weight column name.
+#' @param aggregationDepth depth for treeAggregate (>= 2). If the dimensions of features or the number of partitions
+#'                         are large, this param could be adjusted to a larger size. Default is 2.
+#' @param ... additional arguments passed to the method.
+#' @return \code{spark.logit} returns a fitted logistic regression model
+#' @rdname spark.logit
+#' @aliases spark.logit,SparkDataFrame,formula-method
+#' @name spark.logit
+#' @export
+#' @examples
+#' \dontrun{
+#' sparkR.session()
+#' # binary logistic regression
+#' label <- c(1.0, 1.0, 1.0, 0.0, 0.0)
+#' feature <- c(1.1419053, 0.9194079, -0.9498666, -1.1069903, 0.2809776)
+#' binary_data <- as.data.frame(cbind(label, feature))
+#' binary_df <- suppressWarnings(createDataFrame(binary_data))
```

Comment:
I followed other test examples. If it is not necessary, I can remove all of them as a followup JIRA.